manager/manager.go

```diff
@@ -194,7 +194,12 @@ func New(memoryCache *memory.InMemoryCache, sysfs sysfs.SysFs, houskeepingConfig
 		rawContainerCgroupPathPrefixWhiteList: rawContainerCgroupPathPrefixWhiteList,
 	}
 
-	machineInfo, err := machine.Info(sysfs, fsInfo, inHostNamespace)
+	newManager.perfUncoreCollector, err = perf.NewUncoreCollector(perfEventsFile)
```

Comment:
imo, it'll be good if cAdvisor will log that there is a problem with perf uncore configuration but other machine information will be exposed, could you check this behavior?